app/models/solidus_viabill/payment_method.rb

```diff
@@ -0,0 +1,16 @@
+class SolidusViabill::PaymentMethod < SolidusSupport.payment_method_parent_class
```

Comment:
Can you change the name of this class? Otherwise it will shown as `Payment Method` on the admin settings page.

e.g. http://localhost:3000/admin/payment_methods/1/edit